client/hwlib/tests/test_cert_status.rs

```diff
@@ -1,34 +1,30 @@
 use hwlib::get_certification_status;
-use hwlib::models::rbody::CertificationStatusResponse;
 
 const SERVER_URL: &str = "https://example.com";
 
 #[tokio::test]
 async fn test_get_certification_status_not_seen() {
```

Comment:
We are missing a test that checks against the serialised OpenAPI schema available in the repo.

Reply:
should this test be run for the client changes as well?